scalastyle-config.xml

```diff
@@ -64,7 +64,7 @@
  </check>
  <check level="warning" class="org.scalastyle.scalariform.MethodLengthChecker" enabled="true">
   <parameters>
-   <parameter name="maxLength"><![CDATA[100]]></parameter>
+   <parameter name="maxLength"><![CDATA[150]]></parameter>
```

Comment:
Can you please make a ticket in Jira to fix this issue?

Reply:
Also, I think it's better to disable scalastyle in one particular place instead of changing it globally

Reply:
I thought we talked about it and the decision was to disable scalastyle in one file

Reply:
Yep, that's was my thoughts as well

Reply:
Instead of global allowed method length change, apply method in ProjectSearchService was wrapped in "scalastyle method length off" comment